src/Magnum/Math/Geometry/Intersection.h

```diff
@@ -188,6 +201,89 @@ template<class T> bool boxFrustum(const Range3D<T>& box, const Frustum<T>& frust
     return true;
 }
 
+template<class T> bool sphereFrustum(const Vector3<T>& center, const T radius, const Frustum<T>& frustum) {
+    const T radiusSq = radius*radius;
+
+    for(const Vector4<T>& plane: frustum.planes()) {
+        /* The sphere is in front of one of the frustum planes (normals point
+           outwards) */
+        if(Distance::pointPlaneScaled<T>(center, plane) < -radiusSq)
+            return false;
+    }
+
+    return true;
+}
+
+/**
+@brief Intersection of a point and a cone
+@param p        The point
+@param origin   Origin of the cone
+@param normal   Normal of the cone
+@param angle    Apex angle of the cone
+
+Returns `true` if the point is inside the cone.
+
+@see pointCone(Vector3, Vector3, Vector3, T) @todo: That ref is never going to work...
```

Comment:
Does `@see @ref pointCone(const Vector3<T>&, const Vector3<T>&, T)` work? Yes, it's a bit shitty, but what can we do...

Similarly elsewhere.